### PR TITLE
Add integration with the old App Store 

### PR DESCRIPTION
**Pull request by mariospr**
We use the old App Store to add folders and websites to the desktop, so add the relevant component back.

https://phabricator.endlessm.com/T17236

### CODE REVIEW

File: js/ui/windowManager.js [hunk 1]
```diff
@@ -2136,6 +2136,12 @@ const WindowManager = new Lang.Class({
             }
 
             if (SideComponent.isSideComponentWindow(actor.meta_window) && this._showDesktopOnDestroyDone) {
+                if (!Main.appStore.appLaunched) {
+                    Main.overview.showApps();
+                }
```
Comment (joaquimrocha) [inline js/ui/windowManager.js]: No curly braces on single line bodies. Same above on line 66.

File: js/ui/shellDBus.js [hunk 2]
```diff
@@ -16,6 +17,11 @@ const Main = imports.ui.main;
 const Screenshot = imports.ui.screenshot;
 const ViewSelector = imports.ui.viewSelector;
 
+/* Occurs when an application is installed, meaning added to the desktop's app
```
Comment (joaquimrocha) [inline js/ui/shellDBus.js]: Maybe just saying "Occurs when an application is added to the app grid" is enough?

File: js/ui/windowManager.js (hunk 1 above)
Comment (mariospr) [inline js/ui/windowManager.js]: Ooops! I removed all those from appStore.js, but I clearly forget about this place. I'll fix that

File: js/ui/shellDBus.js (hunk 2 above)
Comment (mariospr) [inline js/ui/shellDBus.js]: This is not my comment (port from older code), but I'm happy to change it

**Pull request merged**